### DISCUSSION TITLE
When trying to sign in get TLS error

### DISCUSSION THREAD

**Discussion by zany2dmax** [zed-industries/zed#16210] [Feature Requests]
Hi, installed zed via homebrew.  seems to work fine except when I try and login, it pops up a browser that says "Authenticating with GitHub, then says Authentication successful.  But the zed window has a pop-up like below:
<img width="481" alt="Screenshot 2024-08-14 at 8 55 30 AM" src="https://github.com/user-attachments/assets/57d24813-0da2-4611-8271-7eef47c19c6e">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
